lib/api/controllers/cli/dump.js

```diff
@@ -91,68 +113,115 @@ function dump (request) {
 
     // dumping std err/out
     console.log('> dumping std err/out');
-    if (process.env.pm_err_log_path) {
-      fs.copySync(
-        path.dirname(process.env.pm_err_log_path),
-        path.join(dumpPath, 'logs'),
-        {
-          preserveTimestamps: true
-        }
-      );
-    }
-    else if (process.env.pm_log_path) {
-      fs.copySync(
-        path.dirname(process.env.pm_log_path),
-        path.join(dumpPath, 'logs'),
-        {
-          preserveTimestamps: true
-        }
-      );
-    }
-    else {
+    if (!process.env.pm_err_log_path && !process.env.pm_log_path) {
       console.log('... no logs found');
+      return resolve();
     }
 
-    // core-dump
-    console.log('> generating core-dump');
-    dumpme(_kuzzle.config.dump.gcore || 'gcore', `${dumpPath}/core`);
-
-    // Gzip the core
-    glob(`${dumpPath}/core*`, (err, res) => {
+    const logDir = path.dirname(process.env.pm_err_log_path || process.env.pm_log_path);
```

Comment:
I am not happy with the way this is done.
Current behaviour is now to take the last created files from pm2 log directory (stderr > stdout) and to compress them.

We may have several cases where this is not suitable imo

* too much dependency on pm2
* no control on what the log directory actually contains
* depends on the logger configuration. For instance, if error logs are sent to syslog, we have no way to retrieve them

I suppose this should be discussed first and would deserve a dedicated implementation out of this PR.